packages/ui/src/services/utils.ts

```diff
@@ -98,4 +97,10 @@ export const getProp = <T extends (Record<string, unknown> | string)> (option: T
   return option
 }
 
-export const generateUuid = () => uuidv4()
+const getRandomString = (stringLength = 4): string => {
+  return Math.random().toString(36).substring(2, stringLength + 2)
+}
+
+export const generateUuid = () => {
```

Comment:
actually it's not uuid because have a different format now, change the naming `generateUuid ` ?

Reply:
`generateUniqueId`?